src/ergw_aaa_config.erl

```diff
@@ -17,6 +17,8 @@
 	 validate_answers/1]).
 -export([to_map/1]).
 
+-include_lib("kernel/include/logger.hrl").
```

Comment:
The `logger` is not used in this module ;-)

Reply:
ok :P